apps/web/src/components/Feed/Posts/PostHeader.tsx

```diff
@@ -71,8 +76,9 @@ export default function PostHeader({ postRef, queryRef }: Props) {
       </HStack>
       <StyledBaseM>
         {post.caption && (
-          <Markdown
-            text={unescape(replaceUrlsWithMarkdownFormat(post.caption))}
+          <ProcessedText
+            text={unescape(post.caption)}
```

Comment:
nit: we no longer need to `unescape`, backend does it for us :)

Reply:
Removed it!